src/Navigation.js

```diff
@@ -0,0 +1,43 @@
+import styled from 'styled-components';
+import { NavLink } from 'react-router-dom';
+
+export default function Navigation() {
+  return (
+    <StyledFooter>
```

Comment:
do you really need the footer around the nav? Isn´t it possible to the style the nav directly like you have done it below?

Reply:
I added the footer around the nav as I want to have a bar on the bottom of the viewport when the app is viewed on the desktop. But the nav should not be spread over the full width. So in this way it looks fine either way - desktop or mobile. Maybe you have another idea to solve that?